<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::QuickInspectorWidget</class>
 <widget class="QWidget" name="GammaRay::QuickInspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QComboBox" name="windowComboBox"/>
   </item>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="previewTreeSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="itemsTab">
        <attribute name="title">
         <string>Items</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="itemTreeSearchLine"/>
         </item>
         <item>
          <widget class="GammaRay::DeferredTreeView" name="itemTreeView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="sceneGraphTab">
        <attribute name="title">
         <string>Scene Graph</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="sgTreeSearchLine"/>
         </item>
         <item>
          <widget class="GammaRay::DeferredTreeView" name="sgTreeView">
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_13">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="GammaRay::PropertyWidget" name="itemPropertyWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_14">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="GammaRay::PropertyWidget" name="sgPropertyWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSaveAsImage">
   <property name="text">
    <string>Save as &amp;Image...</string>
   </property>
   <property name="toolTip">
    <string>Save current frame as image.</string>
   </property>
  </action>
  <action name="actionSaveAsImageWithDecoration">
   <property name="text">
    <string>Save as &amp;Image (with decoration)...</string>
   </property>
   <property name="toolTip">
    <string>Save current frame with decoration rendered as image.</string>
   </property>
  </action>
  <action name="actionAnalyzePainting">
   <property name="text">
    <string>Analyze Painting...</string>
   </property>
   <property name="toolTip">
    <string>Analyzer software renderer paint operations.</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionSlowDownMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable slow down mode</string>
   </property>
   <property name="toolTip">
    <string>Enable slow down mode for animations</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::DeferredTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ui/deferredtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>GammaRay::PropertyWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/propertywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
